env/HERA.env

```diff
@@ -115,14 +115,16 @@ elif [[ "${step}" = "snowanl" ]]; then
     export APRUN_CALCFIMS="${launcher} -n 1"
 
     export NTHREADS_SNOWANL=${NTHREADSmax}
-    export APRUN_SNOWANL="${APRUN_default} --cpus-per-task=${NTHREADS_SNOWANL}"
+    export APRUN_SNOWANL="${APRUN_default} --mem=0 --cpus-per-task=${NTHREADS_SNOWANL}"
```

Comment:
Because without this, jobs run out of memory on Hera. Would you prefer it be added to all machines?

Reply:
What happens if an appropriate value for memory is sent here?  Or does this program have zero memory requirement which seems too good to be true.

Reply:
The only reason I ask here is to reduce work later when we run the code on WCOSS2 and start detecting memory leaks etc.  
Otherwise I have no objection to the "this works" approach.

Reply:
my understanding is 0 means allocate all, and while I understand that is not optimal, we would need to revisit this once JEDI memory management is better resolved. Plus, Hera's days are numbered, so I'd rather do this 'right' on new machines going forward.

Reply:
This is my understanding as well.